modules/rational-ui.el

```diff
@@ -12,7 +12,11 @@
 
 ;;; Code:
 
-(straight-use-package 'all-the-icons)
+(straight-use-package '(all-the-icons
+			:post-build ((when ON-WINDOWS
+				       (warn
+					"%s"
+					"Read the commentary for `rational-ui.el'; on Windows, `all-the-icons-install-fonts' only downloads fonts, they must be installed manually.")))))
```

Comment:
This is the simplest way of _informing_ the user that they may need to take an action, but I'm not sure it is very "rational." The user should be aware of, more or less, what they are doing when they

```emacs-lisp
(require 'rational-ui)
```
It works for the comment on #95, but it makes the code for the module less readable because of the "unnecessary" nature of the commit.

```emacs-lisp
(require 'rational-beware)
```
Might be a module to wrap up things like this, but I'm not sure how that would interact with cases like this package build, where a single form has a system-dependent issue that users _should_ be aware of.

Reply:
Principally,
> [Rational Emacs follows] standard Emacs Lisp patterns so that you can learn by reading the configuration.